smoke-tests-otel-starter/spring-boot-common/src/main/java/io/opentelemetry/spring/smoketest/AbstractOtelSpringStarterSmokeTest.java

```diff
@@ -155,9 +165,17 @@ void shouldSendTelemetry() {
                                         OpenTelemetryAssertions.satisfies(
                                             ServiceIncubatingAttributes.SERVICE_INSTANCE_ID,
                                             AbstractCharSequenceAssert::isNotBlank)))
-                        .hasAttribute(HttpAttributes.HTTP_REQUEST_METHOD, "GET")
-                        .hasAttribute(HttpAttributes.HTTP_RESPONSE_STATUS_CODE, 200L)
-                        .hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping")));
+                        .hasAttributesSatisfying(
+                            attributes ->
+                                OpenTelemetryAssertions.assertThat(attributes)
+                                    .containsEntry(HttpAttributes.HTTP_REQUEST_METHOD, "GET")
+                                    .containsEntry(HttpAttributes.HTTP_RESPONSE_STATUS_CODE, 200L)
+                                    .containsEntry(HttpAttributes.HTTP_ROUTE, "/ping")
+                                    .containsEntry(ServerAttributes.SERVER_ADDRESS, "localhost")
+                                    .containsEntry(ClientAttributes.CLIENT_ADDRESS, "127.0.0.1")
+                                    .hasEntrySatisfying(
+                                        ServerAttributes.SERVER_PORT,
+                                        port -> assertThat(port).isNotZero()))));
```

Comment:
See https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4d6b1cdf3831f16e01e4b4082803e56c2bb11033/instrumentation/jdbc/library/src/test/java/io/opentelemetry/instrumentation/jdbc/datasource/OpenTelemetryDataSourceTest.java#L53 could you write your assert the same way